ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-traffic-engineering.j2

```diff
@@ -0,0 +1,52 @@
+{% if router_traffic_engineering is arista.avd.defined %}
+
+## Router Traffic-Engineering
+
```

Comment:
Please delete this blank line

Reply:
And add that blank line after line 5

Reply:
Whenever we add heading or a table, the blank lines should be on top of that to maintain consistency. For example:

```
{%     if router_traffic_engineering.segment_routing is arista.avd.defined %}

### Segment Routing Summary

- SRTE is enabled.
{%         if router_traffic_engineering.segment_routing.colored_tunnel_rib is arista.avd.defined(true) %}

- system-colored-tunnel-rib is enabled
{%         endif %}
```